src/karmen_frontend/src/components/gcodes/print-gcode-modal.js

```diff
@@ -84,6 +90,19 @@ export const usePrintGcodeModal = ({
               </p>
             )}
 
+            {schedulingPrint && (
+              <div className="modal-content-narrow">
+                <h2 className="modal-subtitle text-center">Scheduling a print...</h2>
+                <Loader image />
+                <p>
+                  Print task is being scheduled to the print queue.
```

Comment:
Your print task is being uploaded to the printer. Please be patient as this can take some time depending on the gcode size and printer network speed.
Do not close this dialog nor your browser window during the upload, or things might <strong>end up badly</strong>!

Reply:
done